internal/presenters/__snapshots__/presenter_sarif_results_pretty_test.snap

```diff
@@ -161,7 +161,7 @@ To view ignored issues, use the --include-ignores option.
 
 
 ╭──────────────────────────────────────────────╮
-│ Test Summary                                 │
+│ [1mTest Summary[0m                                 │
```

Comment:
nitpick: why is that pipe `|` out of line

Reply:
I believe it is tied to the control characters which are rendered in the snapshot but will be hidden when viewed in the terminal.